Tools/ardupilotwaf/boards.py

```diff
@@ -44,7 +44,7 @@ def configure(self, cfg):
                     keys.sort()
                 val = ['%s=%s' % (vk, val[vk]) for vk in keys]
 
-            if k in cfg.env and isinstance(cfg.env, list):
+            if k in cfg.env and isinstance(cfg.env.k, list):
```

Comment:
Oh, thanks for spotting that!

However your fix is also buggy hehehe

It should be `cfg.env[k]` instead of `cfg.env.k`.